code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -23,43 +7,93 @@
 	antag_moodlet = /datum/mood_event/focused
 	antag_hud_type = ANTAG_HUD_TRAITOR
 	antag_hud_name = "traitor"
-	hijack_speed = 0.5 //10 seconds per hijack stage by default
-	var/employer = "The Syndicate"
+	///10 seconds per hijack stage by default
```

Comment:
this is an implementation comment, not a doc comment, and should be `//` 🤓

Reply:
good distinction